src/Bicep.Cli.IntegrationTests/Bicep.Cli.IntegrationTests.csproj

```diff
@@ -10,14 +11,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="coverlet.collector" Version="6.0.3">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
     <PackageReference Include="FluentAssertions" Version="7.0.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.12.0" />
-    <PackageReference Include="MSTest.TestAdapter" Version="3.7.3" />
-    <PackageReference Include="MSTest.TestFramework" Version="3.7.3" />
+    <PackageReference Include="MSTest" Version="$(MSTestVersion)" />
```

Comment:
Do you know if using a variable here will work with Dependabot? The convention we've followed elsewhere in this codebase is to hard-code versions. It doesn't add any process overhead for us, because we rely on Dependabot to bump versions.

Reply:
@anthony-c-martin It works with dependabot. We do it in microsoft/testfx and we get updates